src/index.ts

```diff
@@ -1,134 +1,270 @@
 /**
  * Copyright (c) 2019 mol* contributors, licensed under MIT, See LICENSE file for more info.
  *
- * @author Alexander Rose <alexander.rose@weirdbyte.de>
+ * @author Jesse Liang <jesse.liang@rcsb.org>
  */
 
 import * as argparse from 'argparse'
-import createContext = require('gl')
 import fs = require('fs')
-import { PNG } from 'pngjs'
-import { Canvas3D, Canvas3DParams } from 'molstar/lib/mol-canvas3d/canvas3d';
-import InputObserver from 'molstar/lib/mol-util/input/input-observer';
-import { ColorTheme } from 'molstar/lib/mol-theme/color';
-import { SizeTheme } from 'molstar/lib/mol-theme/size';
-import { CartoonRepresentationProvider } from 'molstar/lib/mol-repr/structure/representation/cartoon';
-import { CIF, CifFrame } from 'molstar/lib/mol-io/reader/cif'
-import { trajectoryFromMmCIF } from 'molstar/lib/mol-model-formats/structure/mmcif';
-import { Model, Structure } from 'molstar/lib/mol-model/structure';
-import { ajaxGet } from 'molstar/lib/mol-util/data-source';
-import { ColorNames } from 'molstar/lib/mol-util/color/tables';
-
-const width = 2048
-const height = 1536
-const gl = createContext(width, height, {
-    alpha: false,
-    antialias: true,
-    depth: true,
-    preserveDrawingBuffer: true
-})
+import { RenderAll, readCifFile, getModels, getID } from './render'
+import { CifFrame, CifBlock } from 'molstar/lib/mol-io/reader/cif';
+import { Model } from 'molstar/lib/mol-model/structure';
 
-const input = InputObserver.create()
-const canvas3d = Canvas3D.create(gl, input, {
-    multiSample: {
-        mode: 'on',
-        sampleLevel: 3
-    },
-    renderer: {
-        ...Canvas3DParams.renderer.defaultValue,
-        lightIntensity: 0,
-        ambientIntensity: 1,
-        backgroundColor: ColorNames.white
-    },
-    postprocessing: {
-        ...Canvas3DParams.postprocessing.defaultValue,
-        occlusionEnable: true,
-        outlineEnable: true
-    }
-})
-canvas3d.animate()
+const parser = new argparse.ArgumentParser({
+    addHelp: true,
+    description: 'Render all models and assemblies of a PDB ID'
+});
+const subparsers = parser.addSubparsers({
+    title: 'subcommands',
+    dest: 'render'
+});
 
```

Comment:
The subparsers share many arguments, please add a helper like `function(parser: argparse.ArgumentParser) { parser.addArgument(); // and more }` to avoid the code duplication.